get/get.go

```diff
@@ -33,38 +33,77 @@ type Cmd struct {
 	ServiceConnection   serviceConnectionCmd  `cmd:"" group:"networking.nine.ch" name:"serviceconnection" aliases:"sc" help:"Get a ServiceConnection."`
 }
 
+type output struct {
+	Format        outputFormat `help:"Configures list output. ${enum}" name:"output" short:"o" enum:"full,no-header,contexts,yaml,stats,json" default:"full"`
+	AllProjects   bool         `help:"apply the get over all projects." short:"A" xor:"watch"`
+	AllNamespaces bool         `help:"apply the get over all namespaces." hidden:"" xor:"watch"`
+	Watch         bool         `help:"Watch resource(s) for changes and print the updated resource." short:"w" xor:"watch"`
+	tabWriter     *tabwriter.Writer
```

Comment:
What about:


```suggestion
     *tabwriter.Writer
```

Reply:
I had this in the beginning but kong did not like having that embedded.

Reply:
Ah, because it is exported. 👍🏼